examples/AvroSpecific/Program.cs

```diff
@@ -48,16 +52,20 @@ static void Main(string[] args)
                 SchemaRegistryUrl = schemaRegistryUrl,
                 // optional schema registry client properties:
                 SchemaRegistryRequestTimeoutMs = 5000,
-                SchemaRegistryMaxCachedSchemas = 10,
-                // optional avro serializer properties:
-                AvroSerializerBufferBytes = 50,
-                AvroSerializerAutoRegisterSchemas = true
+                SchemaRegistryMaxCachedSchemas = 10
             };
 
             var consumerConfig = new ConsumerConfig
             {
                 BootstrapServers = bootstrapServers,
-                GroupId = Guid.NewGuid().ToString()
+                GroupId = "avro-specific-example-group"
+            };
+
+            var avroSerializerConfig = new AvroSerializerConfig
```

Comment:
the configs are specified separately here, but if the user had one giant config with everything, the string property names are such that it's a simple 1 liner to select the relevant config out using LINQ for each component.